Enforce DevFund Commitments with a Legal Charter.rst

```diff
@@ -0,0 +1,110 @@
+-  Requirements & Implementation
+-  The current trademark and sign off issues that just popped up.
+
+
+Motivation
+==========
+
+This proposal is supplemental to any Development Fund Proposal which places or purports to place conditions on how the Electric Coin Company (ECC) and the Zcash Foundation use development funds or take other related off-chain actions (such requirements, Covenants).
+
+For example, the proposal 20% to a 2-of-3 multisig with community-involved governance provides that “[f]unds accruing to the Zcash Development Fund MUST be used only for … technical work directly connected to the various software implementations of the Zcash protocol.” However, once development funding is approved and implemented via a hardfork, there will be no enforcement mechanism to ensure that the ZF and ECC abide by this requirement.
+
+This proposal aims to provide such an enforcement mechanism. If this proposal is adopted, then the ECC and/or ZF, as applicable MUST enter into a legal agreement which would entitle ZCash (ZEC) holders to enforce ECC’s/ZF’s performance of any Covenants. For purposes of this proposal we will refer to the legal agreement as the “DevFund Charter” or “Charter” for short, but it MAY also be styled in other ways–e.g,. as a Constitution, Bylaws, Fund Governance Agreement, etc.
+
+The DevFund Charter should be used to the benefit of all ZEC users, but the DevFund Charter MAY provide that an enforcement action requires the support of the holders of a plurality, majority or supermajority of ZEC. ZEC held by the ZF, ECC and their officers, directors, employees and/or affiliates SHOULD be excluded from the denominator in calculating the requisite plurality, majority or supermajority of ZEC.
+
+A quorum of yet to be decided number of representatives from a number specified groups from which the DevFund Charter MAY provide that an enforcement action requires the support of the assigned representatives of each.  One such mechanism SHOULD be ZEC balance, however this would require a 66% majority or a 85% super majority. (These numbers are not binding and are up for discussion)
+
+It is assumed that the ECC, Foundation or party with a direct conflict of interest WILL identify their vote/signal - which MAY be rejected from the vote.
+
+Legal enforcement MAY occur in a court of law, non-binding mediation or binding arbitration. The DevFund Charter MAY also provide rights to other ZCash community constituencies, such as specified miners or the “Third Entity” contemplated by the 20% to a 2-of-3 multisig with community-involved governance proposal referenced above.
+
+Rationale
+=========
+
+Because ZEC holders do not have specific legal rights against the ECC or ZF, but MAY wish to condition renewed on-chain development funding on the ECC’s or ZF’s agreement to use the development funds in certain ways, ZEC holders SHOULD have the legal right to enforce ECC’s/ZF’s compliance with those agreements.
```

Comment:
This touches on a generally larger problem of how exactly to structure this legal arrangement. Given how difficult the trademark assignment was, the ZIP editors are doubtful that this is possible. To merge this proposal, we'd need to see some proposed specific legal mechanism in one of the relevant jurisdictions.

This objection is partially discussed below, but it suggests crowdsourcing and that isn't really an answer.